test/e2e/statefulset_arbitrary_config/statefulset_arbitrary_config_test.go

```diff
@@ -34,8 +34,8 @@ func TestStatefulSetArbitraryConfig(t *testing.T) {
 
 	mdb.Spec.StatefulSetConfiguration.SpecWrapper.Spec.Template.Spec.Containers[1].ReadinessProbe = &corev1.Probe{TimeoutSeconds: 100}
 
-	serviceName := "database"
-	mdb.Spec.StatefulSetConfiguration.SpecWrapper.Spec.ServiceName = serviceName
+	customServiceName := "database"
+	mdb.Spec.StatefulSetConfiguration.SpecWrapper.Spec.ServiceName = customServiceName
```

Comment:
I changed the parameter to customServiceName to make it clearer to the reader that this is a name that is set by the user. Happy to revert if original parameter name is preferred.